chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/shared_storage_configuration_cluster_cfg.json

```diff
@@ -120,14 +120,14 @@
     ],
     "lustre_devices": [
         {"path_index": 0,
-            "backend_filesystem": "linux"},
+            "backend_filesystem": "zfs"},
```

Comment:
yes , because without it we are not testing the changes in the patch (it was a squashed change originally part of the revert zfs disable patch (PR 41)